rtb/analisi_dei_requisiti_ver0.3.0.typ

```diff
@@ -0,0 +1,325 @@
+viene fornito un Glossario.
+
+La prima occorrenza di un termine definito all'interno del Glossario presente all'interno di un documento viene
+sottolineato e seguito dalla lettera g posta ad apice (e.g. #rifGlossario("termine")).
+
+== Riferimenti
+=== Riferimenti normativi
+- Norme di Progetto
+- Regolamento del progetto didattico\ #formatLink(url: "https://www.math.unipd.it/~tullio/IS-1/2024/Dispense/PD1.pdf")
+- Capitolato C4 (Sync Lab)\ #formatLink(url: "https://www.math.unipd.it/~tullio/IS-1/2024/Progetto/C4.pdf")
+
+=== Riferimenti informativi
+- Glossario
+- T5 - Analisi dei Requisiti\ #formatLink(url: "https://www.math.unipd.it/~tullio/IS-1/2024/Dispense/T05.pdf")
+- P2 - I Diagrammi dei Casi d'Uso\ #formatLink(url: "https://www.math.unipd.it/~rcardin/swea/2022/Diagrammi%20Use%20Case.pdf")
+#pagebreak()
+
+= Descrizione del prodotto
+== Obiettivi del prodotto
+Il prodotto ha come obiettivo quello di fornire un servizio di pubblicità personalizzata integrato in una mappa, accessibile agli utenti di un mezzo a noleggio, che sfrutta la GenAI per la creazione di pubblicità mirate sui singoli utenti. Il gestore del servizio di noleggio avrà accesso ad una _dashboard_ che gli permetterà di monitorare in tempo reale la posizione degli utenti e la visualizzazione degli annunci pubblicitari.
+Per fare ciò l'azienda proponente chiede di simulare lo spostamento degli utenti lungo un percorso, in modo da poter testare il funzionamento del sistema.
+I dati riguardanti gli annunci pubblicitari visualizzati dagli utenti devono essere memorizzati in un _#rifGlossario("database")_ (storicizzazione) in modo da poter essere consultati successivamente dalla _dashboard_, al fine di consentire analisi reportistiche.
+
+Il _benefit_ atteso è quello di rendere le campagne pubblicitarie delle aziende interessate il più efficaci possibili, riducendo la disconnessione tra messaggio e destinatario e aumentando il coinvolgimento di quest'ultimo, migliorando di conseguenza il ROI della campagna stessa.
+
+== Funzionalità e vincoli del prodotto
+Il prodotto dovrà fornire le seguenti funzionalità:
+- *Monitoraggio in tempo reale della posizione degli utenti*: il sistema dovrà essere in grado di mostrare su una mappa interattiva la posizione degli utenti attivi.
+- *Creazione di annunci pubblicitari personalizzati*: il sistema dovrà essere in grado di creare annunci pubblicitari personalizzati per gli utenti, sfruttando i dati di profilazione e la posizione in tempo reale. Quando un utente si trova in prossimità di un punto di interesse il sistema dovrà essere in grado di stimare se l'utente possa essere interessato ai servizi offerti dall'attività e, in tal caso, generare un annuncio pubblicitario personalizzato per quell'utente.
+- *Visualizzazione nella _dashboard_ degli annunci pubblicitari generati*: il sistema dovrà essere in grado di mostrare in tempo reale gli annunci pubblicitari che vengono visualizzati dagli utenti.
```

Comment:
Specificherei meglio la differenza tra le due dashboard:
- [ ] vedo mappa con i sensori che si muovono
- [ ] vedo storico degli annunci
- vedrò probabilemtne altre cose che verranno aggiunte nei prossimi sprint

Reply:
Altra osservazione. Io pensavo onestamente ti mostrare il marker nella mappa e poter visualizzare l'annuncio generato se presente. Se dici di distinguerli, lo faccio. Tutto qui

Reply:
Io quello che avevo capito era: una dashboard con la mappa e i marker in cui possono comparire i messaggi in tempo reale come dici tu, ma poi un'altra dashboard dove vedere la lista degli annunci nel passato (dove dorigo suggeriva di poter filtrarli)

Reply:
Ho modificato come potevo. Mi viene difficile pensare ad altro onestamente su come rifare le frasi